src/Expect.elm

```diff
@@ -278,36 +279,82 @@ notWithin tolerance =
         (\a b -> not <| withinCompare tolerance a b)
 
 
+withinCompare : Float -> Float -> Float -> Bool
 withinCompare tolerance a b =
     let
+        float64minValue =
+            2 ^ -1074
+
         delta =
             abs (a - b)
 
-        -- largest non-infinite value expressible in a 64-bit float
+        -- Largest non-infinite value expressible in a 64-bit float.
```

Comment:
It's probably fine to let the names `float64(min|max)value` describe themselves, but can we move the min down below delta?